core/modules/css/KvInterfaceImplMem.cc

```diff
@@ -136,21 +143,25 @@ KvInterfaceImplMem::set(string const& key, string const& value) {
         throw ReadonlyCss();
     }
 
+    // stored root key is an empty string, but allow "/" too
+    std::string const path(key == "/" ? "" : key);
```

Comment:
Probably worth it to have a initialization function for this as this is getting repeated a bit, or maybe a class with constructor and normalization as those see some repetition as well.

Reply:
Good suggestion, I'll add a function to normalize the path.